models/index.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
-
+require('dotenv').config()
 const fs = require('fs');
 const path = require('path');
 const Sequelize = require('sequelize');
 const basename = path.basename(__filename);
-const config = require(__dirname + '/../config/config').development;
+const config = require(__dirname + '/../config/config')[process.env.DB_TYPE];
```

Comment:
I suggest changing `DB_TYPE` to `NODE_ENV` as you already set the environment with the option `--env test` in the `testDB` script